core/src/main/kotlin/org/eln2/space/Location.kt

```diff
@@ -187,15 +258,27 @@ enum class PlanarFace(val int: Int) {
 	val adjacencies: Array<PlanarFace> get() = PlanarFace.ADJACENCIES[int % 3]
 }
 
+/**
+ * A generic that represents the location of something and its neighbors.
```

Comment:
This could definitely stand to be expounded upon, but I'm alright doing this myself if you want :)